lib/homedoc/views/homedoc.handlebars

```diff
@@ -1,39 +1,44 @@
-[
-  {
-    "rel": "self",
-    "url": "/"
-  },
-  {
-    "label": "Random Password Generator",
-    "rel": "/profile#password_generator",
-    "url": "{{base_url}}/pwd",
-    "model": "{length}",
-    "action": "read",
+{
```

Comment:
I didn't see anywhere that was wrapping this array with the uber top level.  Let me know if i'm missing something